.circleci/create_circleci_config.py

```diff
@@ -211,61 +187,17 @@ def to_dict(self):
             # failure.
             test_command = f"({test_command}) || true"
         else:
-            test_command = f"({test_command} | tee tests_output.txt) || true"
+            test_command = f"({test_command} | tee tests_output.txt)"
```

Comment:
hmm, I don't remember why I added `|| true`, but I believe it's not necessary (at least not anymore)

Reply:
Yes! You did not want to make it fail, but it's actually better to let it fail, but use `when: always` on the condition of the next 3 steps that check the output